src/configuration/useAuthorityLinks.spec.ts

```diff
@@ -0,0 +1,51 @@
+import useAuthorityLinks from "./useAuthorityLinks"
+
+jest.mock("./links.json", () => {
+  return {
+    about: [
+      {
+        url: "firstUrl",
```

Comment:
thoughts on this structure? Should we accommodate that links can be localized also? Maybe there's a query string on the endpoints? And the website where the url is pointing at supports internationalization? @devinjameson @johnschoeman ?

Reply:
All good questions, we dont know the answers to what we need to support in that detail yet.

my guess is that we will start with just a single url, so just localization of the labels.  so we might want to update the structure to accommodate this, but tbh i would start with just defaulting the en and pick up the localization in a follow up commit.

so this structure will likely want to change, but only after we know more so i might start with a simpler one for now such as:
```
about: [
  {
    url: "firstUrl",
    label: "firstLabel"
  },
  {
    url: "secondUrl",
    label: "secondLabel"
  }
]
```
but what you have here looks fine to me.